storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -224,8 +225,7 @@ public static void setDrawerItemOrder(List<String> hiddenItems, List<String> vis
     }
 
     public static List<Integer> getFullNotificationButtons() {
-        String[] buttons = TextUtils.split(
-            prefs.getString(PREF_FULL_NOTIFICATION_BUTTONS,
+        String[] buttons = TextUtils.split(prefs.getString(PREF_FULL_NOTIFICATION_BUTTONS,
```

Comment:
Please revert this change, so we only see the actual code changes in the diff. This ensures that the git history remains easy to read